CHANGELOG.rst

```diff
@@ -7,10 +7,12 @@ xskillscore v0.0.16 (2020-XX-XX)
 
 Internal Changes
 ----------------
+- Add community support documents: ``HOWTOCONTRIBUTE.rst``, issue template and pull request
+  template. `Aaron Spring`_ and `Ray Bell`_
+- Replace ``pandas`` with ``cftime`` in examples and tests. `Aaron Spring`_ and `Ray Bell`_
+- Add coveralls for tests coverage. `Aaron Spring`_ and `Ray Bell`_
+- Add ``isort``, ``doc8`` and ``pre-commit`` for formatting. `Aaron Spring`_ and `Ray Bell`_
```

Comment:
We also use flake8. My PR had the combination of args that worked for isort and flake8. Also climpred

Reply:
Ah ok. I see you use that. Please also list here for completeness although not new.